backend/routes_chemicals.py

```
@@ -30,93 +31,107 @@ def decorated_function(*args, **kwargs):
+        # Start with base query
+        query = Chemical.query
+
+        # Filter by archived status if the column exists
         try:
-            # Get query parameters for filtering
-            category = request.args.get('category')
-            status = request.args.get('status')
-            search = request.args.get('q')
-            show_archived = request.args.get('archived', 'false').lower() == 'true'
+            if not show_archived:
+                query = query.filter(Chemical.is_archived == False)
```

Ruff (0.11.9)

Avoid equality comparisons to `False`; use `if not Chemical.is_archived:` for false checks

Replace with `not Chemical.is_archived`

(E712)